engine/gameobject/src/gameobject/gameobject_props_lua.cpp

```diff
@@ -111,6 +115,32 @@ namespace dmGameObject
             case PROPERTY_TYPE_BOOLEAN:
                 out_var.m_Bool = (bool) lua_toboolean(L, index);
                 return PROPERTY_RESULT_OK;
+            case PROPERTY_TYPE_MATRIX4:
+                {
+                    // JG: Can't we just use memcpy here instead?
```

Comment:
We should.

In `vectormath_aos.h`you'll find:
```
class Matrix4
{
    Vector4 mCol0;
```

Since it's private, the only reference I could find was using:
```
Vector4& v = m[0]; //get col 0
float& f = v[0];
```

Worth a try